src/client/nav/request.js

```diff
@@ -114,6 +114,9 @@ spf.nav.request.send = function(url, opt_options) {
     // the cache response synchronously.
     if (spf.config.get('experimental-sync-response-cache')) {
       handleCache();
+    } else if (spf.config.get('experimental-sync-response-cache-background') &&
+               document.webkitVisibilityState == 'hidden') {
```

Comment:
Yes but limited to webkit.  That said, after chatting, we can apply this to all browsers for testing and see how it goes.

Done.